update priority change (#1601)
cocos-creator/creator-docs: en/ui-system/components/editor/ui-transform.md

        uiTransform.anchorY = 0.5;
    }
}
```

### Deprecation of the priority property

> We deprecated the priority property of the UITransform component in version 3.1, and users can adjust the rendering order by setting the order of the node tree using setSiblingIndex.

Reviewer: ```suggestion
The priority property of the UITransform component was deprecated in version 3.1, and users can adjust the rendering order by setting the order of the node tree using `setSiblingIndex()`.
```

> The priority property of the UITransform component was deprecated in version 3.1, and users can adjust the rendering order by setting the order of the node tree using `setSiblingIndex()`.

Reviewer 2: ```suggestion
The `priority` property of the **UITransform** component was deprecated in v3.1, and users can adjust the rendering order by setting the order of the node tree using `setSiblingIndex()`.
```

> > **Notes**:

Reviewer 2: ```suggestion
**Description of the deprecated `priority` property and the recommended `setSiblingIndex()` method**:

```

> A note on the removal of the priority property and the recommended SiblingIndex property.

Reviewer: ```suggestion
> **Notes**:

> > 1. The priority property on the UITransform component has been removed due to unclear priority and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.

Reviewer 2: ```suggestion
The `priority` property on the **UITransform** component was deprecated in v3.1 due to a lack of clarity and naming conflicts with other properties in the engine. The `priority` property was originally designed to provide a shortcut for the user to sort the node tree, but has no other use in itself and is not related to the meaning of "priority", and actually still adjusts the rendering order by changing the order of the node tree.
```

> The priority property on the UITransform component has been removed due to unclear ideation and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.

Reviewer: > 1. The priority property on the UITransform component has been removed due to unclear ideation and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.

Reviewer: We don't consider __ideation__ to be a positive word. However, I'm not sure what you intend to mean when writing it. Tell me and I can suggest a replacement.

Author: that mean the name of **priority** does not match and clear，which word can I use?

Reviewer: can you use priority?
```suggestion
> 1. The priority property on the UITransform component has been removed due to unclear priority and naming conflicts with other properties in the engine, which was originally designed to provide a shortcut for the user to sort the node tree and has no other use in itself, and is not related to the priority expressed by priority.
```

Author: that's OK,thanks for your help.

> > 2. After removing this property, users can still use the `setSiblingIndex()` method to set the node tree order, but the difference is that there is a default value for priority and the `siblingIndex` of the node is actually the position of the node in the parent node, so after the node tree changes, the This requires the user to know the relative position of the node in the parent node and control it when using the `setSiblingIndex` method in order to get the expected result. The point of the modification here is that the user should not directly equate the `siblingIndex` property with the priority (old) property to understand the use of `siblingIndex`, as their meaning is different.

Reviewer 2: ```suggestion
After deprecating the `priority` property, users can replace it with the `setSiblingIndex()` method, which adjusts the order of the node tree by affecting the `siblingIndex` property of the node. The difference is that the `priority` property has a default value and the `siblingIndex` property of a node is actually the position of the node in its parent node, so the value of the node's `siblingIndex` property will change when the node tree changes. This requires that when using the `setSiblingIndex()` method, the relative position of the node in the parent node is known and controlled in order to obtain the desired result.

> **Note**: the `siblingIndex` property should not be used in the same way as the `priority` (deprecated) property, as they have different meanings. To change the `siblingIndex` property, need to understand and know that it represents the position under the parent node and will change when the node tree changes, and can only be modified by the `setSiblingIndex()` method.
```

> After removing this property, users can still use the setSiblingIndex method to set the node tree order, but the difference is that there is a default value for priority and the siblingIndex of the node is actually the position of the node in the parent node, so after the node tree changes, the This requires the user to know the relative position of the node in the parent node and control it when using the setSiblingIndex method in order to get the expected result. The point of the modification here is that the user should not directly equate the siblingIndex property with the priority (old) property to understand the use of siblingIndex, as their meaning is different.

Reviewer: ```suggestion
> 2. After removing this property, users can still use the `setSiblingIndex()` method to set the node tree order, but the difference is that there is a default value for priority and the `siblingIndex` of the node is actually the position of the node in the parent node, so after the node tree changes, the This requires the user to know the relative position of the node in the parent node and control it when using the `setSiblingIndex` method in order to get the expected result. The point of the modification here is that the user should not directly equate the `siblingIndex` property with the priority (old) property to understand the use of `siblingIndex`, as their meaning is different.
```

> Considering the need for quick sorting of nodes, we will provide a more convenient and quick interface for users to sort nodes in future versions.

Reviewer: ```suggestion
Considering the need for quick sorting of nodes, a more convenient and quick interface for users to sort nodes will be provided in future versions.
```
